website/docs/r/event_orchestration.html.markdown

```diff
@@ -37,9 +37,9 @@ The following arguments are supported:
 The following attributes are exported:
 
 * `id` - The ID of the Event Orchestration.
-* `integration` - An integration for the Event Orchestration.
+* `integration` - A list of integrations for the Event Orchestration.
   * `id` - ID of the integration
-  * `parameters`
+  * `parameters` - A list of parameters
     * `routing_key` - Routing key that routes to this Orchestration.
     * `type` - Type of the routing key. `global` is the default type.
```

Comment:
In the [terraform registry docs](https://registry.terraform.io/providers/PagerDuty/pagerduty/latest/docs/data-sources/event_orchestration), the last two bullets are not displaying as being nested under `parameters`. Any idea how to fix?

Reply:
Thank you for fixing this! Hmm not sure why it's not nested properly, but I think there is a TF doc preview tool, let me see if I can find it.

Reply:
@imjaroiswebdev I'm sorry to be tagging you everywhere today but wasn't it you who showed me a markdown preview tool specifically for Terraform docs? Or am I making things up?